custom_components/tuya_local/devices/klarstein_wonderwall_air.yaml

```diff
@@ -0,0 +1,118 @@
+name: Klarstein Wonderwall Air
+products:
+  - id: cl2wgyhohpjjjuzu
+    name: Wonderwall Art Smart
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+          icon: "mdi:radiator"
+        - dps_val: false
+          value: "off"
+          icon: "mdi:radiator-disabled"
+    - id: 2
+      name: temperature
+      type: integer
+      mapping:
+        - scale: 10
+          step: 10
+      range:
+        min: 0
+        max: 370
+      unit: "\u00b0C"
+    - id: 3
+      name: current_temperature
+      type: integer
+      mapping:
+        - scale: 10
+          step: 10
+      range:
+        min: 0
+        max: 500
+      unit: "\u00b0C"
+    - id: 4
+      type: string
+      mapping:
+        - dps_val: manual
+          value: Manual
+        - dps_val: holiday
+          value: Holiday
+        - dps_val: auto
+          value: Auto
+      name: preset_mode
+secondary_entities:
+  - entity: number
+    name: Timer
+    icon: "mdi:timer-sand"
+    category: config
+    dps:
+      - id: 10
+        type: integer
+        name: value
+        unit: hour
+        range:
+          min: 0
+          max: 24
+  - entity: number
+    name: Countdown
+    icon: "mdi:timer-sand-complete"
+    category: diagnostic
+    dps:
+      - id: 11
+        name: value
+        readonly: true
+        type: integer
+        unit: min
+  - entity: number
+    name: Holiday
+    icon: "mdi:island"
+    category: config
+    dps:
+      - id: 101
+        name: value
+        type: integer
+        unit: day
+        range:
+          min: 0
+          max: 99
+  - entity: select
```

Comment:
Is this really a selector, or is it reporting the result of window open detection?  
"continue" and "close" don't seem self explanatory, is this controlling a window (in which case a cover entity may be better), or is this turning window detection on and off (in which case a switch would be better)?

Reply:
It's turning window detection on/off.
Updated to be better worded but couldn't get a switch to work so it's still a selector.